File: Programming-languages-and-laboratory/Python/Exercises/22_compito_java_20180622/Corso.py
class Corso:
    def __init__(self, codice, tipo, nome, nome_cognome_docente):
        self._codice = codice
        self._tipo = tipo
        self._nome = nome
        self._nome_cognome_docente = nome_cognome_docente

    def get_nome_corso(self):
        return self._nome

    def __str__(self):
        return self._nome + "\t" + str(self._codice)

File: Programming-languages-and-laboratory/Java/Exercises/22_compito_java_2018_06_22/Corso.java
public class Corso {
	protected int codiceCorso;
	protected String tipo, nomeCorso;
	protected String nomeCognomeDocente;
	
	public Corso(int codiceCorso, String tipo, String nomeCorso, String nomeCognomeDocente) {
		this.codiceCorso = codiceCorso;
		this.tipo = tipo;
		this.nomeCorso = nomeCorso;
		this.nomeCognomeDocente = nomeCognomeDocente;
	}
}